README.md

```diff
@@ -196,6 +196,7 @@ Newlines are allowed in the text, as is any other Markdown syntax (both block an
 Callouts can also have a label set within them. The label should be in the format `[label <text>]`.
 The label must be on the first newline after the opening `<$>`.
 The label cannot contain any newlines, but does support inline Markdown syntax.
+The label will be rendered if a label is provided or if the code-block has a toolbar
```

Comment:
I don't think this line should be here -- this content is for the callout plugin?